ft232/libftdi.py

```diff
@@ -24,12 +25,20 @@
 from serial import (FIVEBITS, SIXBITS, SEVENBITS, EIGHTBITS, PARITY_NONE,
                     PARITY_EVEN, PARITY_ODD, STOPBITS_ONE, STOPBITS_TWO)
 
-for lib in ("libftdi.so", "libftdi.so.1"):
-    try:
-        ftdi = c.CDLL(lib)
-        break
-    except OSError:
-        continue
+if platform.system().startswith("Darwin"):
+    for lib in ("libftdi.dylib", "libftdi.dylib.1"):
```

Comment:
You should be able to just add "libftdi.dylib" and "libftdi.dylib.1" to the for loop tuple. There's no need to replicate the entire loop for a different platform.

Reply:
It was made in the same way as in d2xx version.
Files ".dylib" are used only in MacOS and ".so" in Linux, but sure I can combine them into one loop.

Reply:
Done!